python_modules/libraries/dagster-dbt/dagster_dbt_tests/cloud_v2/test_resources.py

```diff
@@ -209,6 +209,10 @@ def my_dbt_cloud_assets(context: AssetExecutionContext, dbt_cloud: DbtCloudWorks
     asset_materialization_events = result.get_asset_materialization_events()
     asset_check_evaluation = result.get_asset_check_evaluations()
 
+    # materializations are successful outputs, asset check evaluations are not outputs
```

Comment:
It feels like there's potentially a missing test layer here. Like; the absence of this should be causing an error when materializing the dbt could assets right? Because there's a required output that didn't get materialized

Reply:
Like; we should be explicitly testing that with mocked data the dbt assets succeed somewhere

Reply:
... which is what you'rre doing a few lines before this? Not sure how I missed that. Disregard